packages/docs/components/button.md

```diff
@@ -25,101 +25,83 @@ links:
 
 ## Anatomy
 
-<Description>
+<Anatomy img="/images/anatomy-button-1.svg" />
 
-Button UI is simple. It consists of typography within a rectangular container.
+<Anatomy img="/images/anatomy-button-2.svg" />
 
-</Description>
+## Behavior
 
-<Anatomy img="/images/anatomy-button-1.svg" />
+<Description>
 
-<Anatomy img="/images/anatomy-button-2.svg" />
+Visually, a Button is a rounded rectangle, wider than it is tall, with a descriptive caption at its center. Users interact with Buttons via both pointer devices and keystrokes.
```

Comment:
> wider than it is tall

This isn't true in all cases though right? Icon only buttons come to mind. Should we reword?

Reply:
Feels weird to me too. @joejezowski-okta I pulled this wording over from Figma; would you be comfortable removing this appositive?

Reply:
yea, I think it feels weird too. Remove!